front/src/applications/operationalStudies/components/Scenario/ScenarioContent.tsx

```diff
@@ -188,16 +202,9 @@ const ScenarioContent = ({
             <div className="scenario-results">
               {collapsedTimetable && (
                 <>
-                  <div className="scenario-timetable-collapsed">
-                    <button
-                      data-testid="timetable-collapse-button"
-                      className="timetable-collapse-button"
-                      type="button"
-                      aria-label={t('toggleTimetable')}
-                      onClick={() => setCollapsedTimetable(false)}
-                    >
-                      <ChevronRight />
-                    </button>
+                  <div
+                    className={cx('scenario-timetable-collapsed', { 'd-none': collapsedTimetable })}
```

Comment:
Hm, one more thing: since this is an a `collapsedTimetable` conditional, `collapsedTimetable` is always true when reaching here. Can we just delete this whole div?

Reply:
Absolutely, didn't realize